integration-cli/docker_cli_run_unix_test.go

```diff
@@ -374,3 +374,21 @@ func (s *DockerSuite) TestRunSwapLessThanMemoryLimit(c *check.C) {
 		c.Fatalf("Expected output to contain %q, not %q", out, expected)
 	}
 }
+
+func (s *DockerSuite) TestRunInvalidCPUShares(c *check.C) {
+	testRequires(c, cpuShare)
+	out, _, err := dockerCmdWithError("run", "--cpu-shares", "1", "busybox", "echo", "test")
+	c.Assert(err, check.NotNil)
```

Comment:
Can you add a `check.Commentf(out)` here? THis way if it fails we can see what happened on the failure.